client/devdocs/sidebar.jsx

```diff
@@ -18,6 +18,16 @@ export default React.createClass( {
 
 	mixins: [ PureRenderMixin ],
 
+	isItemSelected( itemPath, isStrict = true ) {
+		const { path } = this.props;
+
+		if ( isStrict ) {
+			return path === itemPath;
+		}
+
+		return path.indexOf( itemPath ) >= 0;
```

Comment:
I doubt we'd hit an edge case, but should this be either `startsWith` or `=== 0`? We're testing whether the string contains the other, but I believe the path should be at the beginning.

Reply:
👍  e0aa06e6193a81b9891ef4c0d592b79faf7c84ab